config/database.yml

```diff
@@ -14,12 +8,11 @@ development:
   encoding: unicode
   database: spacex
   port: 5432
-  pool: 5
-
+  pool: 100
```

Comment:
A pool size of 100 is quite large for such a simple application. It means that there are 100 concurrent connections possible to the database. I don't think this is necessary and might consume a lot of resources. You can read more about this here: https://devcenter.heroku.com/articles/concurrency-and-database-connections